### ISSUE TITLE
Save NVDA configuration after pressing "OK" on the Welcome Dialog and add a "Cancel" button

### ISSUE THREAD

**Issue by DrSooom** [nvaccess/nvda#8715]
### Path:
NVDA menu » Help » Welcome Dialog...
### Steps to reproduce:
Quote from @michaelDCurran:
> 1. dismiss the welcome dialog without choosing to never show the dialog again.
> 2. configure NVDA not to save settings on exit.
> 3. save their config.
> 4. The next time they start NVDA, the welcome dialog would appear.
> 5. They would choose to never show the dialog again, and then dismiss the welcome dialog.
> 6. Restarting NVDA would cause the Welcome dialog to reappear.
Note: I didn't test this yet exactly that way.
### Actual behaviour:
The NVDA configuration isn't saved after pressing the "OK" button. In NVDA 2018.3 (stable) it will be saved after the Usage Stats Dialog, but only when pressing the buttons "Yes" and "no". If the button "Later" is pressed here, nothing would be saved, even not the settings from the Welcome Dialog.
Expected behaviour:
+ I interpreted [this comment by @michaelDCurran in PR #8708](https://github.com/nvaccess/nvda/pull/8708#issuecomment-418202956) that way, that the NVDA configuration ("nvda.ini") also should be saved directly after pressing the "OK" button on the Welcome Dialog, which of course makes sense if someone disable the checkbox "Save configuration on exit" in the General NVDA Settings.
+ And that's why we now also need a "Cancel" button at this dialog, otherwise there wouldn't be a way closing this window without saving the configuration.
+ Pressing the ESC key will do the same like the "Cancel" button – closing this window without saving.
+ The chapter "4.1.1. Welcome Dialog" in the NVDA User Guide has also to be updated too because of the new behaviour in saving the NVDA configuration.
#### NVDA installed/portable:
Both
#### NVDA Version:
2018.1 (also exists in 2018.3 Beta 3)
### See also:
Issue #8555 and PR #8708

**Comment by seanbudd** [comment]
Closing as won't fix
